platforms/unix/vm/sqUnixExternalPrims.c

```diff
@@ -202,11 +203,7 @@ void *ioLoadModule(char *pluginName)
 
   /* try dlopen()ing LIBRARY_PREFIX<name>LIBRARY_SUFFIX searching only the default locations modulo LD_LIBRARY_PATH et al */
 
-# if defined(HAVE_SNPRINTF)
```

Comment:
because any modern Unix already includes snprintf. All operating systems under the platforms directory have snprintf.